mapdata/WarcraftLegacies/BuffData/Skin/B02Q.json

```
@@ -13,12 +13,12 @@
     {
       "Id": 1885959270,
       "Type": 3,
-      "Value": "Fear Aura"
+      "Value": "|c00FF0000Fear Aura|r"
     },
     {
       "Id": 1700951398,
       "Type": 3,
-      "Value": "This unit is under the fear aura, it has reduced damage."
+      "Value": "This unit is under the effects fear aura; it\u0027s attack damage is currently reduced."
```

```suggestion
      "Value": "This unit is under the effects of Fear Aura; it has reduced attack damage."
```
Sentence structure copied from Devotion Aura